packages/common/src/associations/getAssociationStats.ts

```diff
@@ -53,10 +67,15 @@ export const getAssociationStats = async (
 
     const [{ total: associated }, { total: pending }, { total: requesting }] =
       await Promise.all(
-        queries.map((query: IQuery) => {
-          return hubSearch(query, {
-            requestOptions: context.hubRequestOptions,
-          });
+        queries.map(async (query: IQuery) => {
+          try {
+            const result = await hubSearch(query, {
+              requestOptions: context.hubRequestOptions,
+            });
+            return result;
```

Comment:
Minor but if we are not performing any other operations on the `result` I'd suggest we just do `return await hubSearch(...)` to avoid creating a variable.